js/content/common.js

```diff
@@ -196,16 +196,45 @@ let ExtensionLayer = (function() {
 
     // NOTE: use cautiously!
     // Run script in the context of the current tab
-    self.runInPageContext = function(fun){
-        let script  = document.createElement('script');
-        script.textContent = '(' + fun + ')();';
+    self.runInPageContext = function(fun) {
+        let script  = document.createElement("script");
+        script.textContent = '(' + fun + ")();";
         document.documentElement.appendChild(script);
         script.parentNode.removeChild(script);
     };
 
     return self;
 })();
 
+class Messenger {
+    static sendMessage(msgID, info) {
```

Comment:
There is a "sendMessage" API (in chrome.runtime) that is a separate thing from the "postMessage/onmessage" API (on Window/Worker), should we keep this as `postMessage()`? (Or just `post()`, and then `addListener()`?)

Reply:
`postMessage()` sounds good